workflow/snakemake_rules/common.smk

```diff
@@ -168,6 +186,27 @@ def _get_metadata_by_wildcards(wildcards):
     """
     return _get_metadata_by_build_name(wildcards.build_name)
 
+def _get_clade_recency_for_wildcards(wildcards):
+    # check if builds.yaml contains colors:{build_name}:clade_recency
+    if wildcards.build_name in config["colors"] and 'clade_recency' in config["colors"][wildcards.build_name]:
+        return config["colors"][wildcards.build_name]["clade_recency"]
+    # check if builds.yaml or parameters.yaml contains colors:default:clade_recency
+    elif "colors" in config and "clade_recency" in config["colors"]["default"]:
+        return config["colors"]["default"]["clade_recency"]
+    # else return sensible default value
+    else:
+        return "all"
```

Comment:
_non-blocking: I've already picked on this [elsewhere](https://github.com/nextstrain/ncov/pull/1130#discussion_r1781807136) and don't want implementation details to get in the way of having improved coloring_

In #1131 we settled on "strategy 2" which would translate to:

```py
def _get_clade_recency_for_wildcards(wildcards):
    # check if builds.yaml contains colors:{build_name}:clade_recency
    if wildcards.build_name in config["colors"] and 'clade_recency' in config["colors"][wildcards.build_name]:
        return config["colors"][wildcards.build_name]["clade_recency"]
    # use workflow default
    else:
        return config["colors"]["default"]["clade_recency"]
```

Reply:
Thanks for the catch here. I agree that cleaning up default values to source directly from `parameters.yaml` is the right way to do things. But that said, I think I'd like to tackle this cleanup in a separate PR after this PR and #1130 are merged.